source/features.html.erb

```diff
@@ -0,0 +1,63 @@
+        <ol itemscope itemtype="http://schema.org/ItemList">
+          <li class="sub-navigation__item sub-navigation__item--active" itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem">
+            <a href="/features.html" itemprop="item"><span itemprop="name">Features</span></a>
+          </li>
+          <li class="sub-navigation__item" itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem">
+            <a href="roadmap.html" itemprop="item"><span itemprop="name">Roadmap</span></a>
+          </li>
+        </ol>
+      </nav>
+    </div>
+
+      <div class="column-two-thirds">
+              <h1 class="heading-large">Top features</h1>
+                  <p>GOV.UK Pay uses established Payment Service Providers to deliver your payments service.</p>
+
+                  <p>At no additional cost, we already offer:</p>
+                  <h2 class="heading-small">Payment management</h2>
+                      <ul class="list list-bullet">
+                          <li>take online card payments</li>
+                          <li>issue refunds and email receipts</li>
+                          <li>create reports and search transactions</li>
+                          <li>integrate with existing case management workflows</li>
+                          <li>a trusted GOV.UK-branded experience
+</li>
+                      </ul>
+
+                  <h2 class="heading-small">Digital standards</h2>
+                      <ul class="list list-bullet">
+                          <li>meet the Digital Service Standard</li>
+                          <li>meet accessibility standards</li>
+                          <li>stay secure with full Payment Card Industry Data Security Standard compliance</li>
+                          <li>use our modern RESTful API for straightforward integration</li>
+                      </ul>
+
+                  <h2 class="heading-small">Technical support</h2>
+                      <ul class="list list-bullet">
+                          <li>get 24/7 support from the team at Government Digital Service</li>
+                          <li>read clear documentation on how to integrate with GOV.UK&nbsp;Pay</li>
+                          <li>use a sandbox (test) environment to prototype and try with your service</li>
+                          <li>switch PSPs without additional technical work (technical integration is provider-agnostic)</li>
+                      </ul>
+                  <h2 class="heading-medium">Queries and suggestions</h2>
+                  <p>If you want to find out more about these features, or have some needs that GOV.UK Pay isn&rsquo;t yet meeting, please get in touch with us by emailing <a href="mailto:govuk-pay-support@digital.cabinet-office.gov.uk">govuk-pay-support@digital.cabinet-office.gov.uk</a> or chat with us in our <a href="https://ukgovernmentdigital.slack.com/messages/govuk-pay">cross-government Slack channel</a>.</p>
```

Comment:
You should be able to just use the _’_ character directly rather than `&rsquo;`.